maintainers/scripts/update.py

```diff
@@ -15,6 +15,9 @@
 Order = Literal["arbitrary", "reverse-topological", "topological"]
 
 
+FAKE_ROOT_FOR_INDEPENDENT_DEPENDENCIES = "::fake_root_for_independent_dependencies"
```

Comment:
I like to use `Final` for these globals, since it makes the type checker yell if they are ever assigned. And I'm switching this to a tuple, because of the second suggestion below.

```suggestion
FAKE_ROOTS_FOR_INDEPENDENT_DEPENDENCIES: Final[Sequence[str]] = ("::fake_root_for_independent_dependencies",)
```